lib/scan/admin/engine.py

```diff
@@ -88,8 +87,7 @@ def check(target, user_agent, timeout_sec, log_in_file, language, time_sleep, th
                         )
                     )
                     return 1
-        if version() == 3:
-            content = content.decode("utf8")
+        content = content.decode("utf8")
```

Comment:
does this work? i thought you need to pass "utf-8"

Reply:
That was a typo. Then too it was working
![image](https://user-images.githubusercontent.com/32503192/121942790-7f411380-cd6e-11eb-963a-aadc364e017e.png)